modules/controllers/pool-controller.ts

```diff
@@ -252,18 +250,15 @@ export function PoolController(tracer?: any) {
             await syncTokenPairs(ids, viemClient, routerAddress, chain);
             await updateVolumeAndFees(chain, ids);
 
-            if (hooks) {
-                const poolsWithHooks = await prisma.prismaPool.findMany({
-                    where: { chain, id: { in: ids }, hook: { not: {} } },
-                });
-                await syncHookData(poolsWithHooks, hooks, viemClient, chain);
-                await syncHookReviews();
-            }
-
             // Sync balances for the pools
+            const existingIds = dbPools.map(({ id }) => id);
             const newIds = ids.filter((id) => !existingIds.includes(id));
             await syncBptBalancesFromSubgraph(newIds, vaultSubgraphClient, chain);
 
+            if (hooks && newIds) {
+                await syncHookReviews();
+            }
```

Comment:
why do we need to sync the reviews here and not as part of the syncHookData job?